src/views/ContinuationOut.vue

```diff
@@ -480,8 +476,7 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
 
     // always include continue out code
     // use existing Priority and Waive Fees flags
-    this.updateFilingData('add', FilingCodes.CONTINUATION_OUT, this.staffPaymentData.isPriority,
-      (this.staffPaymentData.option === StaffPaymentOptions.NO_FEE))
+    this.updateFilingData('add', FilingCodes.CONTINUATION_OUT, undefined, true)
```

Comment:
You might be able to use null for the 3rd parameter.

Also please update the comment on line 478.

Reply:
I was thinking of setting it to null but I was looking at the method in the `filing-mixin.ts` file. Line: https://github.com/JazzarKarim/business-filings-ui/blob/78da929d6fd6070e32f30baca3f061199de62eb0/src/mixins/filing-mixin.ts#L51

The examples and parameters describe it as either boolean or undefined. What do you think?

Reply:
I didn't look deep enough at this. Your current code is 👍

Reply:
Looks great!